### ISSUE TITLE
HTML-AAM: Unnamed <form> should map as <div>, not unconditionally as role=“form”

### ISSUE THREAD

**Issue by matatk** [w3c/html-aam#82]
*Background: I have been researching how screen-readers expose landmarks to users, because I work on [a WebExtension that does the same](https://github.com/matatk/landmarks).  This issue has been filed as a spin-off from issues filed in other W3C specs (linked below).*

Assistive technologies do not expose `<form>`s to their users as landmarks unless the `<form>` has an accessible name (via `aria-labelledby` or `aria-label`), for the reasons given below.

1. There are instances in the wild of `<form>` elements wrapping whole pages [1,2,3] - these are not really valid landmarks, so considering them as such would add noise and confusion.

2. When a `<form>` is 'genuine': if it lacks a label, there's not really any useful landmark information there (it's easy enough to navigate to the form/first form control for most users anyway), so adding all unlabelled forms would add too much noise to the landmark navigation.

However, browsers *do* currently expose all `<form>`s as `role=“form”` landmarks, even if they lack an accessible name.  This means all assistive technologies must check for the presence of a name before exposing the landmark to their users.

The aim of this proposal is to simplify the situation so that this check is no longer necessary, and to match the corresponding approach proposed in #79 for `<section>` elements that are unlabelled.

Related issue: #79
Related remote issues:
- https://github.com/w3c/aria/issues/513
- https://github.com/w3c/html-aria/issues/64

---

[1] http://stackoverflow.com/questions/10122375/is-it-semantically-valid-to-wrap-all-pages-tags-inside-the-form-tag-asp-net
[2] http://stackoverflow.com/questions/3038217/why-does-asp-net-wrap-the-page-in-a-form
[3] http://webmasters.stackexchange.com/questions/45578/pages-wrapped-in-a-form-element-what-effect-would-this-have-on-seo-or-other-mac

**Comment by jasonkiss** [comment]
Thanks @matatk. Makes sense to me, but will wait for Core-AAM to resolve [related issue #514](https://github.com/w3c/aria/issues/514) first before updating HTML-AAM.

**Comment by jasonkiss** [comment]
Closing. Duplicates #45.

**Comment by matatk** [comment]
@jasonkiss thanks for your ACK and update; also sorry for the duplicate issue report.
